.gitignore

```
@@ -128,8 +128,8 @@ dmypy.json
 # Pyre type checker
 .pyre/
 
-# NFT wallet
-wallet.json
+# NFT wallets
+.db
```

```suggestion
data/.db
```

maybe these should go in the data folder